reconcile/utils/secret_reader.py

```diff
@@ -32,6 +38,16 @@ def vault_client(self):
             self._vault_client = VaultClient()
         return self._vault_client
 
+    @property
+    def vault_enabled(self):
```

Comment:
Can we add a `bool` return type definition?

Reply:
sure 🤦